salt/utils/verify.py

```diff
@@ -192,106 +189,164 @@ def verify_files(files, user):
     return True
 
 
-def verify_env(dirs, user, permissive=False, pki_dir='', skip_extra=False):
+def verify_env(dirs, user, default_mode=0o750,
+               allow_nonprimary_gid=False, pki_dir='', skip_extra=False):
```

Comment:
I noticed you're calling this function using named arguments for `dirs` and `user`, let's turn these 2 positional arguments into keyword arguments.